### ISSUE TITLE
Next Week Plan (05-07-21)

### ISSUE THREAD

**Issue by bagmk** [bagmk/Quantum_Machine_Learning_Express#11]
Plan for next week
- Write Minimum functioning document 
- what is the physics of the optimizer
- learning basic pytorch

- After the feedback, hyperparameter/ learning rate vs convergence.

- Q: Loss and accuracy s not matching
- Q: Loss is limited by some value
- Q: error warning message

Meeting summary
- the loss/ optimizer function
- Phil 4:13
